api/v1alpha2/mysqlaaduser_conversion.go

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package v1alpha2
+
+func (*MySQLAADUser) Hub() {}
```

Comment:
Is this required for impling an interface? If so should we include an interface assertion in here too?

This file looks real weird to me but it also is just doing what the other conversion files are doing I see...

Reply:
Yeah, this is a marker method required for `v1alpha2.MySQLAADUser` to be the hub type for conversions. Really it's so they can specify the interfaces for conversions. Definitely feels pretty odd. I'll add assertions in both the v1 and v2 files.